dasboards/Alerting_dashboard.ndjson

```diff
@@ -0,0 +1,19 @@
+{"attributes":{"description":"","kibanaSavedObjectMeta":{"searchSourceJSON":"{}"},"title":"Dashboard Menu","uiStateJSON":"{}","version":1,"visState":"{\"aggs\":[],\"params\":{\"fontSize\":12,\"markdown\":\"[Security Overview](#/dashboard/9f302900-8ac7-11ea-b703-43c01853c348) | [Alerting](#/dashboard/ac1078e0-8a32-11ea-8939-89f508ff7909) | [User Explorer](#/dashboard/56d633d0-849a-11ea-b7fb-01bea49d9239) | [User Logons](#/dashboard/10c9e640-860f-11ea-a720-c7a0431f179d) | [Process Explorer](#/dashboard/f8eec760-8ee3-11ea-904c-391ecaa2f2f4) | [Software Overview](#/dashboard/33f0d3b0-8b8a-11ea-b1c6-a5bf39283f12) | [Logging Diagnostics](#/dashboard/ff455d20-9511-11ea-974f-b95a6148fe83)\",\"openLinksInNewTab\":false},\"title\":\"Dashboard Menu\",\"type\":\"markdown\"}"},"coreMigrationVersion":"8.7.1","id":"12735ff0-9396-11ea-b41f-4dc1d87833fe","migrationVersion":{"visualization":"8.5.0"},"references":[],"type":"visualization","updated_at":"2023-05-05T01:00:03.100Z","version":"WzI5NDksMl0="}
```

Comment:
This directory is for you local env right? If so let's remove this from the remote repository. https://stackoverflow.com/a/29277946